mw-webapp/src/utils/auth/handleUserCredentials.ts

```diff
@@ -1,16 +1,29 @@
 import {getRedirectResult} from "firebase/auth";
-import {writeNewUserData} from "src/utils/auth/writeNewUserData";
 import {auth} from "src/firebase";
+import {UserService} from "src/service/UserService";
+
+const newUserBlankWays = {
+  ownWays: [""],
+  favoriteWays: [""],
+  mentoringWays: [""],
+};
 
 export const handleUserCredentials = async () => {
   try {
     const userCredentials = await getRedirectResult(auth);
-    if (userCredentials) {
-      writeNewUserData(
-        {userId: userCredentials.user.uid, userEmail: userCredentials.user.email, userName: userCredentials.user.displayName},
-      );
-    } else {
-      return;
+    if(userCredentials) {
+      if (userCredentials.user.email && userCredentials.user.displayName) {
+        UserService.writeNewUserData(
+          {
+            uuid: userCredentials.user.uid,
```

Comment:
Is it a special using user.uid (instead of user.uuid)?

Reply:
If you write user.uuid you get a tautology - user.user unique id